src/main/java/hudson/plugins/ec2/SlaveTemplate.java

```diff
@@ -126,6 +128,8 @@
 
     public int instanceCap;
 
+    public int minimumNumberOfInstances;
```

Comment:
No need to add more public fields here. That's not a great practice in Java.

Reply:
Indeed. I probably just followed the existing pattern there.

Reply:
Commited a fix